accounting/localizations/getting_started.rst

```diff
@@ -0,0 +1,491 @@
+
+    To be effective in this way, your localization module must contains a *.yml* file containing the following code:
+
+    .. code-block:: xml
+
+        -
+          !python {model: account.chart.template, id: pl_chart_template}: |
+            self[0].try_loading_for_current_company()
+
+    This additional file, usually located in the ``data`` folder, must be loaded at the very last in the ``__manifest__.py`` file.
+
+Configuring my own Chart of Accounts?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+First of all, before I proceed, we need to talk about the templates. A template is a record that allows replica of itself.
+This mechanism is needed when working in multi-companies. For example, the creation of a new account is done using the ``account.account.template`` model.
+However, each company using this chart of accounts will be linked to a replica having ``account.account`` as model.
+So, the templates are never used directly by the company.
+
+Then, when a chart of accounts needs to be installed, all templates dependent of this one will create a replica and link this newly generated record to the company's user.
+It means all such templates must be linked to the chart of accounts in some way. To do so, each one must reference the desired chart of accounts using the ``chart_template_id`` field.
+For this reason, we need to define an instance of the ``account.chart.template`` model before creating its templates.
+
+.. code-block:: xml
+
+    <record id="..." model="account.chart.template">
+        <!-- [Required] Specify the name to display for this CoA. -->
+        <field name="name">...</field>
+
+        <!-- [Required] Specify the currency. E.g. "base.USD". -->
+        <field name="currency_id" ref="..."/>
+
+        <!-- [Optional] Define a parent chart of accounts that will be installed just before this one. -->
+        <field name="parent_id" ref="..."/>
+
+        <!-- [Optional] Define the number of digits of account codes. By default, the value is 6. -->
+        <field name="code_digits">...</field>
+
+        <!-- [Optional] Boolean to show or not this CoA on the list. By default, the CoA is visible.
+         This field is mostly used when this CoA has some children (see parent_id field). -->
+        <field name="visible" eval="..."/>
+
+        <!-- [Optional] Boolean to enable the Anglo-Saxon accounting. By default, this field is False. -->
+        <field name="use_anglo_saxon" eval="..."/>
+
+        <!-- [Optional] Boolean to enable the complete set of taxes. By default, this field is True.
+        This boolean helps you to choose if you want to propose to the user to encode the sale and purchase rates or choose from list of taxes.
+        This last choice assumes that the set of tax defined on this template is complete. -->
+        <field name="complete_tax_set" eval="..."/>
+
+        <!-- [Optional] Specify a prefix of the bank accounts. -->
+        <field name="bank_account_code_prefix">...</field>
```

Comment:
shouldn't be optional... are you sure it is?

Reply:
These fields are not required :-(

`bank_account_code_prefix = fields.Char(string='Prefix of the bank accounts')`
`cash_account_code_prefix = fields.Char(string='Prefix of the main cash accounts')`

Reply:
Should I become a liar and set these fields to [Required]?

Reply:
evaluate the possibility to set these fields as required in the code (is there any localization where it would create a problem?)